src/app/test/api/http/unit/handlers/v1/command_publishing_blocklist_test.py

```diff
@@ -0,0 +1,394 @@
+        self,
+        http_client,
+        base_url,
+        blocklist,
+        app_config_auth_enabled,
+        access_token_user,
+    ):
+        url = f"{base_url}/api/v1/commandpublishingblocklist/"
+        headers = {
+            "Authorization": f"Bearer {access_token_user}",
+            "Content-Type": "application/json",
+        }
+        request = HTTPRequest(url, headers=headers, method="GET")
+
+        response = yield http_client.fetch(request)
+        block_list = json.loads(response.body.decode("utf-8"))
+        assert response.code == 200
+        assert len(block_list["command_publishing_blocklist"]) == 0
+
+    @pytest.mark.gen_test
+    def test_post(self, http_client, base_url, garden, system, blocklist_cleanup):
+        url = f"{base_url}/api/v1/commandpublishingblocklist/"
+
+        headers = {"Content-Type": "application/json"}
+        body = {
+            "command_publishing_blocklist": [
+                {
+                    "namespace": garden.name,
+                    "system": system.name,
+                    "command": command_name,
+                }
+            ]
+        }
+        request = HTTPRequest(
+            url, method="POST", headers=headers, body=json.dumps(body)
+        )
+
+        response = yield http_client.fetch(request)
+
+        assert response.code == 201
+
+    @pytest.mark.gen_test
+    def test_post_400(self, http_client, base_url, garden, system):
+        url = f"{base_url}/api/v1/commandpublishingblocklist/"
+
+        headers = {"Content-Type": "application/json"}
+        body = {
+            "command_publishing_blocklist": [
+                {"system": system.name, "command": command_name}
+            ]
+        }
+        request = HTTPRequest(
+            url, method="POST", headers=headers, body=json.dumps(body)
+        )
+
+        with pytest.raises(HTTPError) as excinfo:
+            yield http_client.fetch(request)
+
+        assert excinfo.value.code == 400
+
+    @pytest.mark.gen_test
+    def test_post_auth(
+        self,
+        http_client,
+        base_url,
+        garden,
+        system,
+        app_config_auth_enabled,
+        access_token_user_admin,
+        blocklist_cleanup,
+    ):
+        url = f"{base_url}/api/v1/commandpublishingblocklist/"
+
+        headers = {
+            "Authorization": f"Bearer {access_token_user_admin}",
+            "Content-Type": "application/json",
+        }
+        body = {
+            "command_publishing_blocklist": [
+                {
+                    "namespace": garden.name,
+                    "system": system.name,
+                    "command": command_name,
+                }
+            ]
+        }
+        request = HTTPRequest(
+            url, method="POST", headers=headers, body=json.dumps(body)
+        )
+
+        response = yield http_client.fetch(request)
+
+        assert response.code == 201
+
+    @pytest.mark.gen_test
+    def test_post_auth_401_no_token(
```

Comment:
You don't actually need to test the no token scenario here.  That's a function of the `AuthorizationHandler` that the blocklist handlers are built on and is tested as part of its test suite.